### PR TITLE
[LA-314] Send data to long running influx for AIO

### PR DESCRIPTION
**Pull request by evrardjp**
K (for leapfrog), N14.1 (for leapfrog + intraseries), N14.0 (for
intraseries), and master should now send metrics to the long
running influxdb instance.

Issue: [LA-314](https://rpc-openstack.atlassian.net/browse/LA-314)

### CODE REVIEW

File: pipeline_steps/common.groovy [hunk 1]
```diff
@@ -56,6 +56,15 @@ def install_ansible_roles(){
   """
 }
 
+def get_influx_credentials(){
+  return [
+    string(
```
Comment (hughsaunders) [inline pipeline_steps/common.groovy]: This can go directly in the withEnv call, unless you see this being used in multiple places?

Comment (evrardjp) [inline pipeline_steps/common.groovy]: If we ever split deploy-rpc- from deploy.sh.

Comment (evrardjp) [inline pipeline_steps/common.groovy]: but I'd rather have your expert advice here.

**Comment by evrardjp** [comment]
This job exercices the change:
https://rpc.jenkins.cit.rackspace.net/job/RPC-AIO_newton141-trusty-deploy-swift-periodic/26/console

File: pipeline_steps/deploy.groovy [hunk 2]
```diff
@@ -21,26 +21,28 @@ def deploy_sh(Map args) {
     stage_name: "Deploy RPC w/ Script",
     stage: {
       environment_vars = args.environment_vars + common.get_deploy_script_env()
-      withEnv(environment_vars) {
-        ansiColor('xterm') {
-          if (!('vm' in args)){
-            dir("/opt/rpc-openstack/") {
+      withCredential(get_influx_credentials()) {
```
Comment (git-harry) [inline pipeline_steps/deploy.groovy]: withCredentials

**Comment by evrardjp** [comment]
https://rpc.jenkins.cit.rackspace.net/job/RPC-AIO_newton141-trusty-deploy-swift-periodic/27/

File: pipeline_steps/deploy.groovy [hunk 3]
```diff
@@ -21,26 +21,28 @@ def deploy_sh(Map args) {
     stage_name: "Deploy RPC w/ Script",
     stage: {
       environment_vars = args.environment_vars + common.get_deploy_script_env()
-      withEnv(environment_vars) {
-        ansiColor('xterm') {
-          if (!('vm' in args)){
-            dir("/opt/rpc-openstack/") {
+      withCredentials(get_influx_credentials()) {
```
Comment (odyssey4me) [inline pipeline_steps/deploy.groovy]: I'd suggest changing this to:

```
withCredentials([
  string(
    credentialsId: "INFLUX_IP",
    variable: "INFLUX_IP"
  )]) {
```

**Comment by evrardjp** [comment]
https://rpc.jenkins.cit.rackspace.net/job/RPC-AIO_newton141-trusty-deploy-swift-periodic/28/console

**Comment by evrardjp** [comment]
The job seems to run but let's see if it succeeds all the way.

**Comment by evrardjp** [comment]
success!

**Pull request merged**